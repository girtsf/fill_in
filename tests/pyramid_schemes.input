.........######.#.........................########..
.........#....#.#..........#..............#...#.....
.........#..#####..........#..........######..#.....
.......#####..#.####.########.........#...#...######
.........#....#..#.#.......#..........#...#...#.....
.........#....#..#.#..#....#...#....#.#.#####.......
.........#....#..#.#..#....#...#...####...#.........
.........#....#..#....#..########..#..#...#.........
......#####...#..#....#....#...#...#......#####.....
.......#.........#....#.....#########.........#.....
.....#########...#..#########..#..............#.....
.......#.....#......#.#......#######C.........#.....
.......#.....########.#........#..............#.....

AIRFIELD
AMUSE
ARABITOL
DAYLIGHT
DEER
DYER
EDIFIERS
ELEGIACAL
ESTO
EVENT
IGNORE
IONIC
IRISH
IRRITANTS
ISSUES
LASE
LE
LEASE
LEGIONARY
LID
MARIA
MEETS
OPAL
PISCINAE
PRECLUDES
PROTISTA
SAFARI
SATURNIC
SET
SILLIEST
SOLITAIRE
STREEP
SVENGALIS
SWARM
SY
TREMBLED
