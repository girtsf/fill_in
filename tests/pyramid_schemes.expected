.........######.#.........................########..
.........#....#.#..........#..............#...#.....
.........#..#####..........#..........######..#.....
.......#####..#.####.########.........#...#...######
.........#....#..#.#.......#..........#...#...#.....
.........#....#..#.#..#....#...#....#.#.#####.......
.........#....#..#.#..#....#...#...####...#.........
.........#....#..#....#..########..#..#...#.........
......#####...#..#....#....#...#...#......#####.....
.......#.........#....#.....#########.........#.....
.....#########...#..#########..#..............#.....
.......#.....#......#.#......#######C.........#.....
.......#.....########.#........#..............#.....

.........SAFARI.O.........................EDIFIERS..
.........V....R.P..........P..............L...R.....
.........E..MARIA..........I..........ISSUES..I.....
.......IONIC..I.LASE.SILLIEST.........G...G...STREEP
.........G....T..I.S.......C..........N...I...H.....
.........A....A..R.T..D....I...P....L.O.SWARM.......
.........L....N..F.O..A....N...R...DEER...C.........
.........I....T..I....Y..ARABITOL..Y..E...A.........
......AMUSE...S..E....L....E...T...E......LEASE.....
.......E.........L....I.....SOLITAIRE.........V.....
.....PRECLUDES...D..LEGIONARY..S..............E.....
.......T.....E......I.H......SATURNIC.........N.....
.......S.....TREMBLED.T........A..............T.....
